src/components/CreditCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+function CreditCard(props) {
+    const {type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props;
+  return (
+    <div className="CreditCard">
+      <p>
+        <b>Type:</b> {type}
+      </p>
+      <p>
+        <b>Number:</b> {number}
```

Comment:
Here you are just missing rendering multiple "*" followed by the last 4 numbers of the card